Src/BootCamp.Chapter/TextTable.cs

```diff
@@ -38,7 +38,84 @@ public static class TextTable
         /// </summary>
         public static string Build(string message, int padding)
         {
-            return "";
+            if (message.Length == 0)
+            {
+                return ""; 
+            }
+            // make a array so I can find the longest word. 
+
+            var arrayOfWords = message.Split(Environment.NewLine);
+
+            // Find length of longest word 
+           var longestWord = ""; 
+
+            if (arrayOfWords.Length == 1)
+            {
+                longestWord = arrayOfWords[0]; 
+            }
+            else
+            {
+                switch (arrayOfWords[0].CompareTo(arrayOfWords[1]))
+                {
+                    case -1:
+                        longestWord = arrayOfWords[1]; 
+                        break;
+                    default:
+                        longestWord = arrayOfWords[0]; 
+                        break;
+                }
+            }
+
+            // print table 
+
+            // print header
+            var table = "+"; 
+            table += $"{String.Empty.PadRight(longestWord.Length + 2 * padding , '-')}"; 
+            table += "+"; 
+            table +=$"{Environment.NewLine}";
+
+            // print empty lines
+            for (int i = 0; i < padding; i++)
+            {
+                table += "|"; 
+                table += $"{ String.Empty.PadRight(longestWord.Length + 2 * padding, ' ')}";
+                table += "|"; 
+                table += $"{Environment.NewLine}";
+            }
+
+            // print text 
+
+            foreach (var word in arrayOfWords)
+            {
+                table += "|";
+                table += String.Empty.PadRight(padding, ' ');
+                table += word;  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use a StringBuilder instead.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4929752)

Reply:
solved